### PR TITLE
docs: restructuring the readme, moving things into categories for better flow

### PR DESCRIPTION
**Pull request by NeonSpork**
This is a restructuring of the readme layout. I did my best to separate things into useful categories, and assign things appropriately.

This pull request is followup from the discussion [#2395](https://github.com/appwrite/appwrite/discussions/2395) in the main `appwrite` repository.

### CODE REVIEW

**Review by PineappleIOnic — COMMENTED** [review]
This looks great! It will definitely improve the general readability of the runtimes and shine more light onto the resources the community has made for appwrite

**Review by PineappleIOnic — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -61,94 +76,208 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
...
-## Tutorials 
-* [Getting Started for Web](https://appwrite.io/docs/getting-started-for-web) (Official)
-* [Getting Started for Flutter](https://appwrite.io/docs/getting-started-for-flutter) (Official)
-* [Getting Started for Server](https://appwrite.io/docs/getting-started-for-server) (Official)
-* [Getting Started for Android](https://appwrite.io/docs/getting-started-for-android) (Official)
+## Hacktoberfest 2021
+* [What is Appwrite and how to contribute for Hacktoberfest](https://dev.to/mishmanners/four-projects-to-contribute-to-this-hacktoberfest-pep)
+* [Appwrite, giving it a shot this Hacktoberfest ‘21](https://medium.com/@anand26shweta/appwrite-giving-it-a-shot-ad1ef79ae44f)
+
+## Announcements and official blog posts
```
Comment (adityaoberai) [inline README.md:83]:
```suggestion
## Announcements and Official Blog Posts
```

File: README.md [hunk 2]
```diff
@@ -61,94 +76,208 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
...
 * [Sending a Custom Welcome Email Using Appwrite Functions and Mailgun](https://dev.to/appwrite/sending-a-custom-welcome-email-using-appwrite-functions-and-mailgun-225a) (Official)
 * [Create an Appwrite File Backup Function Using the Dropbox API](https://dev.to/appwrite/create-an-appwrite-file-backup-function-using-the-dropbox-api-2pfo) (Official)
 * [Learn How to Create and Run Appwrite Functions With Dart](https://dev.to/appwrite/learn-how-to-create-and-run-appwrite-functions-with-dart-5668) (Official)
-* [Announcing the Appwrite SDK for Dart](https://dev.to/appwrite/announcing-appwrite-sdk-for-dart-3ehc) (Official)
-* [Announcing the Appwrite CLI 🚀](https://dev.to/appwrite/announcing-the-appwrite-cli-4m68) (Official)
 * [Learn How to Disable ClamAV in Your Appwrite Stack and Reduce Memory Usage](https://dev.to/appwrite/learn-how-to-disable-clamav-in-your-appwrite-stack-and-reduce-memory-usage-2e37) (Official)
 * [Appwrite Releases a Native Svelte SDK for Its Open-Source BaaS](https://dev.to/appwrite/appwrite-releases-a-native-svelte-sdk-for-its-open-source-baas-5g0g) (Official)
+## Learning resources
```
Comment (adityaoberai) [inline README.md:92]:
```suggestion
## Learning Resources 
```

File: README.md [hunk 3]
```diff
@@ -61,94 +76,208 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
...
 * [Sending a Custom Welcome Email Using Appwrite Functions and Mailgun](https://dev.to/appwrite/sending-a-custom-welcome-email-using-appwrite-functions-and-mailgun-225a) (Official)
 * [Create an Appwrite File Backup Function Using the Dropbox API](https://dev.to/appwrite/create-an-appwrite-file-backup-function-using-the-dropbox-api-2pfo) (Official)
 * [Learn How to Create and Run Appwrite Functions With Dart](https://dev.to/appwrite/learn-how-to-create-and-run-appwrite-functions-with-dart-5668) (Official)
-* [Announcing the Appwrite SDK for Dart](https://dev.to/appwrite/announcing-appwrite-sdk-for-dart-3ehc) (Official)
-* [Announcing the Appwrite CLI 🚀](https://dev.to/appwrite/announcing-the-appwrite-cli-4m68) (Official)
 * [Learn How to Disable ClamAV in Your Appwrite Stack and Reduce Memory Usage](https://dev.to/appwrite/learn-how-to-disable-clamav-in-your-appwrite-stack-and-reduce-memory-usage-2e37) (Official)
 * [Appwrite Releases a Native Svelte SDK for Its Open-Source BaaS](https://dev.to/appwrite/appwrite-releases-a-native-svelte-sdk-for-its-open-source-baas-5g0g) (Official)
+## Learning resources 
+### Getting started
```
Comment (adityaoberai) [inline README.md:93]:
```suggestion
### Getting Started
```

File: README.md [hunk 4]
```diff
@@ -61,94 +76,208 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
...
+* [Appwrite - macOS Installation and Debugging Guide](https://dev.to/kritikash18/appwrite-macos-installation-and-debugging-guide-28fm)
+* [Appwrite: All you need to know](https://krishguptadev.hashnode.dev/appwrite-all-you-need-to-know)
 * [Set up the backend for your first app in the no-code way using Appwrite!](https://medium.com/@vidushig2020/set-up-the-backend-for-your-first-app-in-the-no-code-way-6052b348d045)
-* [Appwrites Avatars service](https://dev.to/ashish4arora/appwrites-avatars-service-4gae)
-* [Appwrite + Appcelerator Titanium: A step by step guide](https://github.com/m1ga/from_zero_to_app/blob/master/appwrite_app.md)
+* [How to setup Appwrite on Ubuntu](https://dev.to/noviicee/how-to-setup-appwrite-on-ubuntu-3j67)
+* [Open Source Backend Server - Appwrite](https://dev.to/hardikchopra242/open-source-backend-server-appwrite-48b9)
+* [Python in Appwrite ](https://dev.to/robimez/python-in-appwrite-378h)
+
+### Setup, customization and deploying
```
Comment (adityaoberai) [inline README.md:115]:
```suggestion
### Setup, Customization and Deployment
```

File: README.md [hunk 5]
```diff
@@ -61,94 +76,208 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
...
+* [Run Your Appwrite Backend Server With Any SMTP Provider for Max Email Deliverability](https://medium.com/@eldadfux/learn-how-to-run-your-appwrite-backend-server-with-any-smtp-provider-for-max-email-deliverability-c6ab2c3efec8?source=friends_link&sk=c4ad7bffb3952547cfb89137da581310)
+* [Tutorial on how to use an SMTP provider to manage Appwrite end servers to enhance email delivery](https://knowlegdeninja.blogspot.com/2021/10/tutorial-on-how-to-use-smtp-provider-to.html)
+* [Deploy Appwrite to AWS using Terraform and Ansible](https://dev.to/rizkyrajitha/deploy-appwrite-to-aws-using-terraform-and-ansible-4af)
+* [Setup Appwrite on Linode](https://imknight.com/setup-appwrite-on-linode/)
+* [Manage Appwrite data with Appsmith](https://imknight.com/manage-appwrite-data-with-appsmith/)
+* [Appwrite + Zapier = Magic](https://dev.to/ruthlessruler/appwrite-zapier-magic-12gk)
+* [Import sample data to Appwrite through Integromat](https://imknight.com/import-data-to-appwrite-through-integromat/)
+* [Integrate Adalo with Appwrite](https://imknight.com/integrate-adalo-with-appwrite/)
+
+### Web development
```
Comment (adityaoberai) [inline README.md:130]:
```suggestion
### Web Development
```

File: README.md [hunk 6]
```diff
@@ -61,94 +76,208 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
...
 * [Getting Started with Appwrite in NextJS by Building An App](https://dev.to/qwe123coder/getting-started-with-appwrite-in-nestjs-22gk)
-* [Using Appwrite Locale Service in your Next Project](https://dev.to/drishtipeshwani/using-appwrite-locale-service-in-your-next-project-3pcm)
-* [Introduction to Appwrite's Avatar API](https://drishtip.hashnode.dev/introduction-to-appwrites-avatar-api)
-* [Dart in Appwrite](https://dev.to/oshi36/dart-in-appwrite-47e)
 * [Integrate Appwrite Storage API with React](https://dev.to/harisarang/integrate-appwrite-storage-api-with-react-5dg3)
-* [Appwrite Storage Service](https://dev.to/yashp1210/appwrites-storage-service-ned)
-* [Appwrite + Zapier = Magic](https://dev.to/ruthlessruler/appwrite-zapier-magic-12gk)
 * [How To Generate and Store Map Previews in Ruby with Appwrite](https://dev.to/oliverpham/how-to-generate-and-store-map-previews-in-ruby-with-appwrite-43ha)
+
+### Mobile development
```
Comment (adityaoberai) [inline README.md:146]:
```suggestion
### Mobile Development
```

File: README.md [hunk 7]
```diff
@@ -61,94 +76,208 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
...
+* [Appwrite the best backend for mobile apps and its features](https://qr.ae/pGxRuF)
+* [Appwrite secure open-source backend server for Flutter example app](https://dev.to/netfirms/appwrite-and-flutter-example-app-42ce)
+* [How setup Appwrite for Android with Retrofit ( Direct HTTP Request )](https://medium.com/@developersm13/how-to-setup-appwrite-io-for-android-with-retrofit-e6271d770cad)
+* [How to use Appwrite's Flutter SDK](https://dev.to/yashp1210/appwrite-for-flutter-58kb)
+* [Introduction to Appwrite Cloud Functions with Android and Kotlin](https://hardiksachan.hashnode.dev/introduction-to-appwrite-cloud-functions-with-android-and-kotlin)
+* [Dart in Appwrite](https://dev.to/oshi36/dart-in-appwrite-47e)
+* [Appwrite + Appcelerator Titanium: A step by step guide](https://github.com/m1ga/from_zero_to_app/blob/master/appwrite_app.md)
+
+### Security and authentication
```
Comment (adityaoberai) [inline README.md:155]:
```suggestion
### Security and Authentication
```

File: README.md [hunk 8]
```diff
@@ -61,94 +76,208 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
...
+* [Restricting Access to Your Appwrite Console](https://medium.com/appwrite-io/you-can-now-restrict-access-to-your-appwrite-console-b8b447885289?source=friends_link&sk=95b78cf75ff633e0f32b8a76ea619b08)
+* [User authentication with React.js and Appwrite](https://medium.com/@e.wambugu192/user-authentication-with-react-js-and-appwrite-30c9096e92b2)
+* [Full List of Appwrite Environment Variables](https://appwrite.io/docs/environment-variables)
+* [How to Use Magic URL Login with Appwrite](https://muthuannamalai.tech/how-to-use-magic-url-login-with-appwrite)
+* [How to use Appwrite on React Native](https://dev.to/fernandoamz/appwrite-react-native-48cn)
+
+### Appwrite services
```
Comment (adityaoberai) [inline README.md:162]:
```suggestion
### Appwrite Services
```

File: README.md [hunk 9]
```diff
@@ -61,94 +76,208 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
...
+* **Gradely 2 - Grade Calculator** - A grade calulator app where you can monitor and store your grades, built with Appwrite and Flutter.  [Source Code](https://github.com/generalxhd/Gradely2)
+
+* **Fitness Challenge App with Appwrite** -  Create fitness challenges, invite friends, sync your health data from Apple Health/Google Fit and see who's winning. [Source Code](https://github.com/gireeshp/fitness_challenge) [Blog Post](https://medium.com/@gireeshputhumana/building-the-fitness-challenge-app-with-appwrite-367286790f64)
+
+* **Quit Smoking App with Flutter and Appwrite** - A simple Counting app with user auth  [Source Code](https://github.com/Jonathanvanhaaften/flutter_appwrite_1) 
+
+### Godot
+* **Godot + Appwrite** - A demo app to showcase Godot's integration with Appwrite. [Source Code](https://github.com/lohanidamodar/godot-appwrite)
+
+### golang
```
Comment (adityaoberai) [inline README.md:223]:
```suggestion
### Golang
```

**Review by adityaoberai — CHANGES_REQUESTED** [review]
Love the work you've done here @NeonSpork 😄 
I've made some suggestions on the formatting of the title.
Do give them a glance once as well.

**Review by NeonSpork — COMMENTED** [review]
@adityaoberai  I've made the changes to the capitalization of the sub-headers as requested, and additionally capitalized the respective entries in the links to content section at the top, 🙂

File: README.md [hunk 10]
```diff
@@ -3,12 +3,27 @@
 ![logo](images/logo.png "Logo")
 
 ## Table of Contents:
-
-  - [Tutorials](#tutorials)
+  - [Announcements and Official Blog Posts](#announcements-and-official-blog-posts)
+  - [Learning Resources](#learning-resources)
+    - [Getting Started](#getting-started)
+    - [Setup, Customization and Deploying](#setup,-customization-and-deploying)
```
Comment (adityaoberai) [inline README.md:9]:
```suggestion
    - [Setup, Customization and Deploying](#setup-customization-and-deployment)
```

File: README.md [hunk 11]
```diff
@@ -3,12 +3,27 @@
...
+    - [Web Development](#web-development)
+    - [Mobile Development](#mobile-development)
+    - [Security and Authorization](#security-and-authorization)
+    - [Appwrite Services](#security-and-authorization)
```
Comment (adityaoberai) [inline README.md:13]:
```suggestion
    - [Appwrite Services](#appwrite-services)
```

**Review by adityaoberai — CHANGES_REQUESTED** [review]
Just noticed a couple of anchor links that needed to be fixed here in the Table of Contents. Otherwise, looks like we're good!

File: README.md [hunk 12]
```diff
@@ -3,12 +3,27 @@
...
 ## Table of Contents:
-
-  - [Tutorials](#tutorials)
+  - [Announcements and Official Blog Posts](#announcements-and-official-blog-posts)
+  - [Learning Resources](#learning-resources)
+    - [Getting Started](#getting-started)
+    - [Setup, Customization and Deploying](#setup,-customization-and-deploying)
+    - [Web Development](#web-development)
+    - [Mobile Development](#mobile-development)
+    - [Security and Authorization](#security-and-authorization)
```
Comment (adityaoberai) [inline README.md:12]:
```suggestion
    - [Security and Authentication](#security-and-authentication)
```

**Review by adityaoberai — CHANGES_REQUESTED** [review]

**Comment by NeonSpork** [comment]
> Just noticed a couple of anchor links that needed to be fixed here in the Table of Contents. Otherwise, looks like we're good!

Oops, I missed those! Good spot. Also agreed that it should be changed to authentication.

**Review by adityaoberai — APPROVED** [review]
Changes look good to merge now 👍

**Comment by adityaoberai** [comment]
THANK YOU! All changes merged 🥳 

As a way of saying thank you, we would also love to invite you to **join the Appwrite organization** on GitHub. Please share your GitHub username with us on Discord so that we can add you as a contributor to our organization.

Once added, you can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite membership badge will show up on your personal GitHub profile.

**Pull request merged**